src/main/java/net/neoforged/neoforge/network/PacketDistributor.java

```diff
@@ -19,26 +19,17 @@
 import net.minecraft.server.level.ServerPlayer;
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.level.ChunkPos;
-import net.neoforged.neoforge.internal.NeoForgeProxy;
 import net.neoforged.neoforge.server.ServerLifecycleHooks;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Means to distribute packets in various ways
+ * <p>
+ * Serverbound payloads must be sent via {@code ClientPacketDistributor#sendToServer()}
```

Comment:
_must_ is a bit of an overstatement. I'd rather use "can". Also, you forgot the final . of the sentence. ^^